apps/authz/kinds/resourcepermission.cue

```diff
@@ -0,0 +1,24 @@
+package kinds
+
+import (
+	"github.com/grafana/grafana/apps/authz/kinds/v0alpha1"
+)
+
+resourcepermission: {
+	kind:       "ResourcePermission"
+	pluralName: "ResourcePermissions"
+	current:    "v0alpha1"
+
+	codegen: {
+		ts: { enabled: false }
```

Comment:
Just a sanity check, are we planning on generating TS code for the frontend clients at some point in the future?

Reply:
Probably for role bindings and managed permissions at least.

Reply:
It's up to you, I just wanted to double-check! 🙂